### PR TITLE
Dawntrail branch initial setup

### PR DESCRIPTION
**Pull request by AkairyuGestalter**
This PR contains a bunch of start-of-expac housekeeping and setup to prepare for Dawntrail.

Summary of changes:
- Add github config for the dawntrail branch
- Update the home page background image to one for the Arcadion raid series
- Add VPR and PCT to the job data and stub out their parser code
- Squash and remove the Endwalker action and status layer files
- Remove Endwalker patch-specific code from analysis code
- Purge the contributor and changelog files to give us a clean slate for Dawntrail contributions
- Set core and all jobs to an unsupported patch range
- Remove Endwalker patches from the patch list, and add 7.0 patch date
- Minor bugfix for the home page job support summary to keep jobs that aren't done from showing as supported once core gets marked clean for 7.0
- Other miscellanous cleanup and i18n re-extract

### CODE REVIEW

File: src/data/ACTIONS/root/PLD.ts
Comment (Toastdeib) [inline src/data/ACTIONS/root/PLD.ts:1]: Leaving this as a file comment because I apparently can't comment on lines that weren't modified - it looks like we de-comboed the Confiteor combo in `patch6.3.ts`, but the steps in the flattened file still have their combo properties. Is that correct, or should they be removed? I'm assuming correct since the actual tooltips still list the combo actions, but I wanted to confirm.

File: src/parser/jobs/dnc/changelog.tsx
Comment (Toastdeib) [inline src/parser/jobs/dnc/changelog.tsx:8]: lel

File: src/parser/jobs/mch/modules/Tincture.tsx
Comment (Toastdeib) [inline src/parser/jobs/mch/modules/Tincture.tsx:1]: These seem like functional changes rather than patch squishing; any reason for including them here? I'm assuming it doesn't actually change the logic but I'm curious.

File: src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap
Comment (Toastdeib) [inline src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap:42]: No clue what this file is; do you know what the change here means?

**Review by Toastdeib — COMMENTED** [review]
Couple of small comments, but I think this looks good to go for the most part. I'm mainly curious about the one I left on the MCH Tincture file. I'm happy to approve and merge it once you've had a chance to respond.

File: src/parser/jobs/mch/modules/Tincture.tsx
Comment (AkairyuGestalter) [reply src/parser/jobs/mch/modules/Tincture.tsx:1]: It's pulling in the changes from #1946 so we don't lose them. That PR merged in after I branched this, and figured I'd grab it since this was still under review

File: src/parser/jobs/dnc/changelog.tsx
Comment (AkairyuGestalter) [reply src/parser/jobs/dnc/changelog.tsx:8]: It bothered me since last expac and just didn't do anything about it then 🙃

File: src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap
Comment (AkairyuGestalter) [reply src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap:42]: It's some unit test file that broke now that the old Trick Attack raid buff status was fully gone from the data files. I had to switch it to the Mug raid buff status to get it to pass.

File: src/data/ACTIONS/root/PLD.ts
Comment (AkairyuGestalter) [reply src/data/ACTIONS/root/PLD.ts:1]: Yeah, I'm not sure what happened there, but they're definitely still a combo in-game. I re-checked it today and every step of Confetti -> Swordsx3 gets combo broken if you do your normal combo in between

File: src/parser/jobs/mch/modules/Tincture.tsx
Comment (Toastdeib) [reply src/parser/jobs/mch/modules/Tincture.tsx:1]: Are the `endwalker` and `dawntrail` branches fully diverged now? I don't actually know our branching structure for the major ones like these two, but I assumed we would want to merge the changes in that PR up from `endwalker` into `dawntrail` rather than duplicate them.

Comment (Toastdeib) [reply src/parser/jobs/mch/modules/Tincture.tsx:1]: After a discussion with ack on Discord, sounds like we can keep these without it being much of a problem.

**Review by Toastdeib — APPROVED** [review]
Marking this as approved. I'll let it sit a little longer before merging in case anyone else wants to chime in, but if not, I'll push the button this afternoon.

**Pull request merged**
